src/runtime/plan/suggestion.ts

```diff
@@ -20,27 +20,31 @@ import {Search} from '../recipe/search.js';
 
 export class Plan {
   serialization: string;
-  particles: {name: string}[] = [];
+  particles: {name: string, connections: {}[]}[] = [];
```

Comment:
might be worthwhile to define as a type?

Reply:
I'm thinking it might be better to move this into the recipe/Recipe.ts and make it a formal JSON friendly Recipe format. But i want to discuss it with @shans first, so let's leave it for now?

Reply:
sgtm